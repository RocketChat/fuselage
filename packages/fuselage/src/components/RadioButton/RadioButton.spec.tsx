import React from 'react';
import ReactDOM from 'react-dom';

import { RadioButton } from '../..';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<RadioButton />, div);
  ReactDOM.unmountComponentAtNode(div);
});
