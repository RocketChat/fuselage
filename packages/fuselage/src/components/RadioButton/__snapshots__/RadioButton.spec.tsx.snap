// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RadioButton Rendering] renders Checked without crashing 1`] = `
<body>
  <div>
    <label
      class="rcx-box rcx-box--full rcx-label  rcx-box rcx-box--full rcx-radio-button"
    >
      <input
        aria-label="Decorator Label"
        checked=""
        class="rcx-box rcx-box--full rcx-radio-button__input"
        type="radio"
      />
      <i
        aria-hidden="true"
        class="rcx-box rcx-box--full rcx-radio-button__fake"
      />
    </label>
  </div>
</body>
`;

exports[`[RadioButton Rendering] renders Default without crashing 1`] = `
<body>
  <div>
    <label
      class="rcx-box rcx-box--full rcx-label  rcx-box rcx-box--full rcx-radio-button"
    >
      <input
        aria-label="Decorator Label"
        class="rcx-box rcx-box--full rcx-radio-button__input"
        type="radio"
      />
      <i
        aria-hidden="true"
        class="rcx-box rcx-box--full rcx-radio-button__fake"
      />
    </label>
  </div>
</body>
`;

exports[`[RadioButton Rendering] renders Disabled without crashing 1`] = `
<body>
  <div>
    <label
      class="rcx-box rcx-box--full rcx-label  rcx-box rcx-box--full rcx-radio-button"
    >
      <input
        aria-label="Decorator Label"
        class="rcx-box rcx-box--full rcx-radio-button__input"
        disabled=""
        type="radio"
      />
      <i
        aria-hidden="true"
        class="rcx-box rcx-box--full rcx-radio-button__fake"
      />
    </label>
  </div>
</body>
`;
